test/parse.js

```diff
@@ -19,7 +19,12 @@ var Path = require('path');
 // http://mxr.mozilla.org/mozilla-central/source/js/src/tests/ecma_5/JSON/
 
 var dirsPath = Path.resolve(__dirname, 'parse-cases');
-var dirs = FS.readdirSync(dirsPath);
+var dirs = FS.readdirSync(dirsPath).filter(function(fn) {
+  if (fn.startsWith('.')) {
```

Comment:
@jordanbtucker I don't need credit - especially for these little things. I've just been moving around the code y'all have already written.